lib/cartopy/tests/mpl/test_mpl_integration.py

```diff
@@ -112,7 +111,7 @@ def test_global_scatter_wrap_no_transform():
 @pytest.mark.natural_earth
 @pytest.mark.mpl_image_compare(
     filename='global_hexbin_wrap.png',
-    tolerance=2 if MPL_VERSION < parse_version('3.2') else 0.5)
+    tolerance=0.5)
```

Comment:
The default is 0.5, so this can be dropped.

Reply:
Thanks, and apologies for the CI thrashing here. The conda-forge testing labels were not in all of the releases, so had to pin to a specific version and make sure `matplotlib-base` is from the testing label too.

Reply:
@greglucas Apologies on that. Probably those releases pre-date cartopy using them. It should be much easier keeping testing up-to-date nowadays.

Reply:
No problem at all, that actually wasn't too bad to diagnose compared to some of the other flaky failures we seem to have here ;)